modular_robot_physical/revolve2/modular_robot_physical/_physical_robot_config.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import pickle
+from dataclasses import dataclass
+
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.body.base import ActiveHinge
+
+
+@dataclass
+class PhysicalRobotConfig:
+    """The configuration for running a physical robot."""
+
+    modular_robot: ModularRobot
+    """The Modular Robot Object."""
+    hinge_mapping: dict[ActiveHinge, int]  # which servo is triggered by which hinge.
+    simulation_time: int  # in seconds.
+    control_frequency: int
+    inverse_servos: dict[int, bool] | None = None
+    """Inverse servos according to ActiveHinge objects or pin IDs."""
+
+    @staticmethod
+    def from_pickle(pickled_object: bytes | str) -> PhysicalRobotConfig:
```

Comment:
I would remove this function altogether and just leave the pickling and unpickling up to the user. `pickle.loads` and `pickle.dumps` should already provide the behaviour we need for when we want to write/read to a file

Reply:
I see. Let's leave that out for now and add it when we add the remote control